### DISCUSSION TITLE
How to create a level with map elements like Town[1-10] only from an openDrive map?

### DISCUSSION THREAD

**Discussion by Utopialjp** [carla-simulator/carla#5412] [Q&A]
CARLA version: 0.9.13
Platform/OS: Linux/Ubuntu

I have an openDrive map, but no RoadRunner. Referring to document [OpenDRIVE standalone mode](https://carla.readthedocs.io/en/latest/adv_opendrive/), now I can open level OpenDriveMap.umap. When I play in UE Editor, the actor OpenDriveGenerator will spawn some actors about map elements like this.
![image](https://user-images.githubusercontent.com/23400855/167066950-58e3f069-aba7-4e2c-b6a2-312c205e8c56.png)
But when play ends, these actors are gone. How can I save these actors into a Level like Town[1-10], so that I can see them directly when I open the level without OpenDriveGenerator?

**Comment by fangchaooo** [comment]
I also want to kown how to save or edit it? It seems running in play runtime

**Comment by Reezzz702** [comment]
I have the same problem too. 
Recently, I tried to ingest OpenStreetMap into Carla following the tutorial from [https://carla.readthedocs.io/en/latest/tuto_G_openstreetmap/](url), but it seems that the content can only be seen in play mode, after terminate play mode it would turn back to Town[1-10]. How can I edit it if i want to create a new town by ingesting OpenStreetMap?
